docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/upgrade-guides/6-0-0-upgrade-guide/index.md

```diff
@@ -101,25 +101,30 @@ After upgrading RDB Loader, you might find out that events or entities with some
 
 To avoid this, the offending older Iglu schemas must be patched to align with the latest.
 
-You can use the latest version of `igluctl` to do this:
+You can use the latest version of `igluctl` to do this.
+
+To illustrate, let's say we have `1-0-0` and `1-0-1` that differs on one field only:
+```
+* version `1-0-0`: `{ "type": "integer", "maximum": 100 }` - translates to `SMALLINT`. All is going good.
+* version `1-0-1`: `{ "type": "integer", "maximum": 1000000 }` - breaking change as it translates to `INT` and loader can't
+  migrate.
+```
+Since `1-0-1` had breaking change, migration had been performed manually in warehouse previously because, loader couldn't migrate it due to breaking change. After manual migration, events with `1-0-1` schema would be loaded with pre-6.0.0 RDB Loader versions. However, after upgrading RDB Loader to 6.0.0, events with `1-0-1` schema will start to land in the recovery table.
```

Comment:
```suggestion
Since version `1-0-1` had a breaking change, loading must have broken with an older version of RDB Loader. To fix that, the user must have updated the warehouse manually (changing the column type to `INT`). After this manual intervention, events with the `1-0-1` schema would have been loaded successfully with older versions of RDB Loader. However, after an upgrade to RDB Loader 6.0.0, events with the `1-0-1` schema will start to land in the recovery table.
```